src/core/bridge.ts

```diff
@@ -19,6 +27,23 @@ export interface BridgeCallback {
   (err?: Error, res?: Results): void;
 }
 
+export class BridgeTimeout extends Error {
+  constructor (err: string) {
+    super(err);
+  }
+}
+
+export class BridgeError extends Error {
+  constructor (statusText: string, public status: number, public data: any) {
+    super(statusText);
```

Comment:
we lost overall coverage because of these lines, can we add a simple test to this